core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -873,6 +873,22 @@ class GroupCoordinatorTest {
     assertEquals(Errors.FENCED_INSTANCE_ID, invalidHeartbeatResult)
   }
 
+  @Test
+  def shouldGetDifferentStaticMemberIdBasedOnTimestamp() {
+    staticMembersJoinAndRebalance(leaderInstanceId, followerInstanceId)
+
+    val timeAdvance = 1
+    for (_ <- 1 to 20) {
```

Comment:
20 seems like overkill. Maybe two times is enough.

Reply:
Yea, will reduce it to 5.